src/components/Chart/Chart.stories.tsx

```diff
@@ -135,6 +136,7 @@ const referenceAreas: ReferenceArea<number>[] = [
     ],
     color: '#ffeed6',
     tickColor: '#b58b00',
+    dashed: true,
```

Comment:
```suggestion
    dashed: false,
```
I think it's important to show all options at storybook